Tasks/Common/azure-arm-rest/AzureResourceFilter.ts

```diff
@@ -0,0 +1,33 @@
+import msRestAzure = require('./azure-arm-common');
+import webClient = require('./webClient');
+import azureServiceClient = require('./AzureServiceClient');
+import tl = require('vsts-task-lib/task');
+import Q = require('q');
+import { AzureResourceFilterManagementClient } from './azure-arm-resource-filter';
+import { AzureEndpoint } from './azureModels';
+
+export class AzureResourceFilterController {
+    private _azureResourceFilterManagementClient: AzureResourceFilterManagementClient;
+    private _endpoint: AzureEndpoint;
+
+    constructor(endpoint) {
```

Comment:
consider providing an overload or a converter method directly in the base client class